applications/ChimeraApplication/ChimeraApplication.py

```diff
@@ -3,6 +3,7 @@
 
 # Application dependent names and paths
 from KratosMultiphysics import _ImportApplication
+from KratosFluidDynamicsApplication import *
```

Comment:
```suggestion
import KratosMultiphysics.FluidDynamicsApplication
```

Otherwise you have the FD-stuffs also in the Chimera-namespace

Reply:
Yes. I made that change to see if that actually is the problem. It seems it is. 
But honestly I do not like and am not completely see if it is to be done like that. 

@roigcarlo  Can you have also a look here ? 

The point is there is an import of a fluid dynamics app in the __init__.py py of ChimeraApp (of course ChimApp depends on FDApp). This is what I do not like, that too to treat a failure which happens at the very end of CI during import (@philbucher  is it so ?)

Reply:
It should be done like this. I thought you already did this/I mentioned it somewhere

Chimera links against Fluid, so if you do not import the symbols before importing Chimera then it will fail.
Many other apps do that too, PotentialFlow (also depends on Fluid), Trilinos (depends on MPI-Core)

Reply:
The point is that you should be able to do only 
~~~py
import KratosMultiphysics.ChimeraApplication
~~~
without it failing (this is what happened in the CI bcs alphabetically Chimera comes before Fluid). How the things are loaded is an implementational detail the user should not have to know abt

In a usual workflow this is not a problem in most cases since the dependencies (Fluid in this case) are othen imported before somewhere else